Unable to lanuch native browser

When attempting to download a modserver trough the FTBLauncher i get the following error log
`[11:58:04] [ERROR] Desktop.checkActionSupport:225->OSUtils.browse:374: Could not open link: java.lang.UnsupportedOperationException: The BROWSE action is not supported on the current platform!`

A quick check in OSUtils.java reveal that when checking for desktop integration (isDesktopSupported) no check for the browse action itself is performed ([isSupported(action)](http://docs.oracle.com/javase/7/docs/api/java/awt/Desktop.html#isSupported%28java.awt.Desktop.Action%29))

For the record my system is a non-standard unix desktop; the xdg-open fallback works just fine but apparently isDesktopSupported returns true despite that the browse action is unavailable.

--- a/src/main/java/net/ftb/util/OSUtils.java
+++ b/src/main/java/net/ftb/util/OSUtils.java
@@ -480,7 +480,7 @@ public class OSUtils {
      */
     public static void browse (String url) {
         try {
-            if (Desktop.isDesktopSupported()) {
+            if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Action.BROWSE)) {
                 Desktop.getDesktop().browse(new URI(url));
             } else if (getCurrentOS() == OS.UNIX && (new File("/usr/bin/xdg-open").exists() || new File("/usr/local/bin/xdg-open").exists())) {
                 // Work-around to support non-GNOME Linux desktop environments with xdg-open installed
@@ -502,7 +502,7 @@ public class OSUtils {
             return;
         }
         try {
-            if (Desktop.isDesktopSupported()) {
+            if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Action.OPEN)) {
                 Desktop.getDesktop().open(path);
             } else if (getCurrentOS() == OS.UNIX) {
                 // Work-around to support non-GNOME Linux desktop environments with xdg-open installed

